web_widget_text_markdown/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,6 @@
+This module adds a new widget for text field in form view on Odoo:
+
+- In readonly mode, it uses text contents to parse and render them to html markdown syntax.
+- In write mode, use [bootstrap-markdown][1]
+
+[1]: https://github.com/toopay/bootstrap-markdown    "bootstrap-markdown"
```

Comment:
```suggestion
[1]: `bootstrap-markdown <https://github.com/toopay/bootstrap-markdown>`_
```

Reply:
This is a markdown syntax and i don't get the same result with this changes :/

Reply:
The README file is rst format not md

Reply:
Oops! done, thx! me confused with available markup languages :S